### DISCUSSION TITLE
Token of thanks to all admins of this collection!

### DISCUSSION THREAD

**Discussion by zriyansh** [prik-k/awesome-OS#4] [Announcements]
People like you are an inspiration to all. 
Thank you for this generous act of open sourcing these projects. I feel privileged to make a collection of some of the awesome and cool things on the Internet so that others can find these projects useful and can contribute.

Honorable mentions: [In no particular order]

@Rajaniraiyn

@PiyushSuthar 

@blueedgetechno 

@virtualvivek 

@ShizukuIchi  

@1j01  

@felixrieseberg  

@wobsoriano  

@arturbien

@gabrielbull  

@PuruVJ  

@Renovamen 

@guyariely   

@vivek9patel   

@captbaritone

@remigallego

@durasj

@OXOYO

@padraigfl

@lolstring

@emupedia

@kevin-shannon

@1000hz

@benwiley4000

@rjanjic

@scottschiller

@tpenguinltg

@cubiclesoft

@smore-inc

@jseidelin

@basicallydan

@moltenform

@khang-nd

@botoxparty

@jdan

@kristopolous

@soroushchehresa

@jsdf

@STRd6

@os-js

@FriendUPCloud

@MineAndCraft12

@shmuelhizmi

@nathansmith

@coolwanglu

@timhow38

@davidedc

@Joeyonng

@trungdq88

@heathervv

@GoodManWEN

@jckdm

@Yalgie

@Manthee1

@D-Antonelli

@edwardpayton

@nbriz

@DustinBrett

@koas

@Fingel

@marcmascarell

@pkage

@tholman

@victorqribeiro

@Gamithra

@tojrobinson

@zcaceres

@afterstep

@pippinbarr

@arturbien

@AshKyd

@grassmunk

@kalcaddle

@sirredbeard

@juddy

@melkael

@paultopia

@v-y-l

@jhhoward

@ssshake

@SerenityOS

@partiellkorrekt

@michitaro

special mention 
@copy

**Comment by zriyansh** [comment]
@GoodManWEN

@jckdm

@Yalgie

@Manthee1

@D-Antonelli

@edwardpayton

@nbriz

@DustinBrett

@koas

@Fingel

@marcmascarell

@pkage

@tholman

@victorqribeiro

@Gamithra

@tojrobinson

@zcaceres

@afterstep

@pippinbarr

@arturbien

@AshKyd

@grassmunk

@kalcaddle

@sirredbeard

@juddy

@melkael

@paultopia

@v-y-l

@jhhoward

@ssshake

@SerenityOS

@partiellkorrekt

@michitaro



special mention 
@copy

**Reply by DustinBrett** [reply]
Thanks. Cool list you got here, reminds me of https://github.com/syxanash/awesome-web-desktops. I am working on a v2 of my desktop that will hopefully be quite a bit more feature packed than the old one. It's such a fun side project. I originally got inspired from people like https://github.com/SerenityOS/serenity & https://github.com/MineAndCraft12/AaronOS which I see on your list.

**Reply by zriyansh** [reply]
Can't wait to see v2 of xOS. Indeed it took a bit of work to get all these projects under a roof. I hope people get inspiration of projects listed here and make something cool!
Cheers
